zoo/services/forms.py

```diff
@@ -60,6 +60,14 @@ class Meta:
         }
 
 
+class LinkForm(forms.ModelForm):
+    class Meta:
+        model = models.Link
+        fields = ["name", "url", "icon"]
+
+    labels = {"name": "Name", "url": "URL", "icon": "Emoji"}
```

Comment:
it's not emoji, so it might not need the labels at all in the end
```suggestion
    labels = {"name": "Name", "url": "URL", "icon": "Icon"}
```